camp/abolotski/XBoxToTalon/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
What would be better is if we have a bit to describe the file here.  We have a block used in Yogi2 ... take a look, can you find a better block?

Reply:
changed to the yogi2 block